### ISSUE TITLE
[FEATURE] tapping username / icon in chat 

### ISSUE THREAD

**Issue by theotheroracle** [commetchat/commet#294]
i think it would be nice if you could tap on the username or icon of someone in a chat to open their profile !

**Comment by Airyzz** [comment]
I agree, been meaning to do this for a long time! happy for you to work on it :D
